pkg/serviceaccounts/oauthclient/oauthclientregistry.go

```diff
@@ -234,30 +255,41 @@ func (a *saOAuthClientAdapter) GetClient(ctx kapi.Context, name string) (*oautha
 		RedirectURIs: sets.NewString(redirectURIs...).List(),
 		GrantMethod:  a.grantMethod,
 	}
+
+	// TODO: is this safe to tell - could leak Route info?
+	a.recorder.Eventf(sa, kapi.EventTypeNormal, "OAuthAllRedirectURIs", "Has the following redirectURIs: %v", saClient.RedirectURIs)
+
```

Comment:
You're sending one event per oauth login to the master?  That's... probably going to eat into our write budget.